src/pybind/mgr/orchestrator/module.py

```diff
@@ -115,6 +117,8 @@ def to_yaml_n(objs: list) -> list:
         if many:
             return yaml.dump_all(to_yaml(copy), default_flow_style=False)
         return yaml.dump(to_yaml(copy), default_flow_style=False)
+    elif format == Format.xml or format == Format.xml_pretty:
+        raise OrchestratorError(f"format '{format.name}' is not implemented.")
```

Comment:
```suggestion
        raise OrchestratorError(f"format '{format.name}' is not implemented for 'ceph orch' commands.")
```
NIT, technically this format is implemented in other areas of ceph. It's ceph orch commands specifically that we mean here.